app/views/comments/_edit.html.erb

```diff
@@ -1,15 +1,50 @@
-<form id="c<%= comment.id %>edit" style="display:none;" class="well" action="/comment/update/<%= comment.id %>">
```

Comment:
I think this may be better as a separate template. Could you name it `_answer.html.erb` and switch at a higher level?